server/views/layouts/main.hbs

```diff
@@ -3,9 +3,11 @@
   <head>
     <meta charset="utf-8">
     <title> Rudimentalna drustvena mreza </title>
+		<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5/jquery.min.js"></script>
```

Comment:
Ne treba ti ovdje jquery, osobito ne ova stara verzija. jQuery ces instalirati preko `npm install` i importati u `assets/javascript/application.js` i onda ce sve to zavrsiti u `/public/app.js` kada pokrenes `npm build`